models/src/main/java/com/vimeo/networking2/enums/GroupForumsPrivacyValue.kt

```diff
@@ -0,0 +1,13 @@
+package com.vimeo.networking2.enums
+
+/**
+ * Type of group forums.
+ */
+enum class GroupForumsPrivacyValue {
+
+    /**
+     * Only moderators can comment on the group.
+     */
+    MODS
```

Comment:
Should `UNKNOWN` types be added to all these enums that were added here? I don't see them mentioned in the PR description so I'm not sure what the plans are for them.